workspace/src/main/openapi/workspace.yaml

```diff
@@ -797,11 +748,146 @@ components:
+        useDedicatedEventHubNamespace:
+          type: boolean
+          default: false
+          description: Set this property to true to use a dedicated Azure Event Hub Namespace for this Workspace.
+        dedicatedEventHubSasKeyName:
+          type: string
+          description: the Dedicated Event Hub SAS key name, default to RootManageSharedAccessKey
+        dedicatedEventHubAuthenticationStrategy:
+          type: string
+          description: the Event Hub authentication strategy
+        sendScenarioRunToEventHub:
+          type: boolean
+          default: true
+          description: default setting for all Scenarios and Run Templates
+        sendScenarioMetadataToEventHub:
+          type: boolean
+          default: false
+          description: Set this property to false to not send scenario metada
+        datasetCopy:
+          type: boolean
+          default: true
+          description: Activate the copy of dataset on scenario creation
+        security:
+          $ref: '#/components/schemas/WorkspaceSecurity'
       required:
         - key
         - name
         - solution
+
+    WorkspaceUpdateRequest:
+      type: object
+      description: Request object for updating a workspace
+      properties:
+        key:
+          type: string
+          description: technical key for resource name convention and version grouping. Must be unique
+        name:
+          type: string
+          description: Workspace name
+        description:
+          type: string
+          description: the Workspace description
+        tags:
+          type: array
+          description: the list of tags
+          items:
+            type: string
+        solution:
+          $ref: '#/components/schemas/WorkspaceSolution'
+        webApp:
+          $ref: '#/components/schemas/WorkspaceWebApp'
+        sendInputToDataWarehouse:
+          type: boolean
+          description: default setting for all Scenarios and Run Templates
+        useDedicatedEventHubNamespace:
+          type: boolean
+          description: Set this property to true to use a dedicated Azure Event Hub Namespace
+        dedicatedEventHubSasKeyName:
+          type: string
+          description: the Dedicated Event Hub SAS key name
+        dedicatedEventHubAuthenticationStrategy:
+          type: string
+          description: the Event Hub authentication strategy
+        sendScenarioRunToEventHub:
+          type: boolean
+          description: default setting for all Scenarios and Run Templates
+        sendScenarioMetadataToEventHub:
+          type: boolean
+          description: Set this property to false to not send scenario metadata
+        datasetCopy:
+          type: boolean
+          description: Activate the copy of dataset on scenario creation
+      required:
```

Comment:
Why are any of the fields required for the update request? Why do I have to provide the existing key/name if I only want to change the tags for example?